gatsby-node.js

```diff
@@ -95,29 +95,35 @@ exports.createPages = async ({ graphql, actions }) => {
           return node;
         });
 
-      const hasMultiplePrograms = allPagesData.filter(node => node.type === 'pages.ProgramPage');
-
-      if (hasMultiplePrograms && hasMultiplePrograms.length > 1) {
-        // Create landing page
+      // If we spot an enterprise page, create it, and do not create programs pages
+      const isEnterprise = allPagesData && allPagesData[0].pageData.pageType === 'pages.EnterprisePage';
```

Comment:
nit: Might want to create a variable for the first page in `allPagesData` to avoid having to retrieve `allPagesData[0].pageData` twice.

```js
const firstPageData = allPagesData && allPagesData[0].pageData;
const isEnterprise = firstPageData && firstPageData.pageType === 'pages.EnterprisePage';

if (isEnterprise) {
  createPage({
    // ...
    context: firstPageData,
  });
}
```